scripts/get_installed_golang_version.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+if go version > /dev/null 2>&1 ; then
```

Comment:
it would be nice if you could extract the go version information in a single call rather than making two calls.
( but it looks like it would work correct either way )

Reply:
@tsachiherman Updated to make a single call to go version.